package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@condenast/cn-react-slick",
-  "version": "0.14.5",
+  "version": "0.14.5-cn.1",
```

Comment:
That is correct and we have a PR in progress to be proposed against the issue in the OSS repo.